data/compute_magic_prime.py

```
@@ -0,0 +1,39 @@
+import json, math, random, sys, time, shutil, os, string, re, fileinput
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import

(F401)